sql/information_schema/routines_table.go

```diff
@@ -136,42 +138,41 @@ func routinesRowIter(ctx *Context, c Catalog, p map[string][]*plan.Procedure) (R
-				procedure.Name,             // routine_name NOT NULL
-				"PROCEDURE",                // routine_type NOT NULL
-				"",                         // data_type
-				nil,                        // character_maximum_length
-				nil,                        // character_octet_length
-				nil,                        // numeric_precision
-				nil,                        // numeric_scale
-				nil,                        // datetime_precision
-				nil,                        // character_set_name
-				nil,                        // collation_name
-				"",                         // dtd_identifier
-				"SQL",                      // routine_body NOT NULL
-				procedure.Body.String(),    // routine_definition
-				nil,                        // external_name
-				"SQL",                      // external_language NOT NULL
-				"SQL",                      // parameter_style NOT NULL
-				isDeterministic,            // is_deterministic NOT NULL
-				"",                         // sql_data_access NOT NULL
-				nil,                        // sql_path
-				securityType,               // security_type NOT NULL
-				procedure.CreatedAt.UTC(),  // created NOT NULL
-				procedure.ModifiedAt.UTC(), // last_altered NOT NULL
-				sqlMode,                    // sql_mode NOT NULL
-				procedure.Comment,          // routine_comment NOT NULL
-				procedure.Definer,          // definer NOT NULL
-				characterSetClient,         // character_set_client NOT NULL
-				collationConnection,        // collation_connection NOT NULL
-				collationServer,            // database_collation NOT NULL
+				procedure.Name,                  // specific_name NOT NULL
+				"def",                           // routine_catalog
+				db,                              // routine_schema // TODO: get correct db associated to this procedure
+				procedure.Name,                  // routine_name NOT NULL
+				"PROCEDURE",                     // routine_type NOT NULL
+				"",                              // data_type
+				nil,                             // character_maximum_length
+				nil,                             // character_octet_length
+				nil,                             // numeric_precision
+				nil,                             // numeric_scale
+				nil,                             // datetime_precision
+				nil,                             // character_set_name
+				nil,                             // collation_name
+				"",                              // dtd_identifier
+				"SQL",                           // routine_body NOT NULL
+				procedure.CreateProcedureString, // routine_definition  // body.String() gives query plan
+				nil,                             // external_name
+				"SQL",                           // external_language NOT NULL
+				"SQL",                           // parameter_style NOT NULL
+				isDeterministic,                 // is_deterministic NOT NULL
+				nil,                             // sql_data_access NOT NULL
```

Comment:
Should we return something different since this column is defined as `NOT NULL` (according to the comment on this line at least). Similar comment for `sql_mode` below, too. Assuming those column definitions in the columns are correct, seems like we should return something even if we have a todo to go back and return the right value.